packages/core/cms/faststore/content-types.json

```diff
@@ -99,8 +99,27 @@
   {
     "id": "pdp",
     "name": "Product Page",
-    "isSingleton": true,
-    "configurationSchemaSets": []
+    "configurationSchemaSets": [
+      {
+        "name": "Settings",
+        "configurations": [
+          {
+            "name": "template",
+            "schema": {
+              "title": "Template",
+              "type": "object",
+              "properties": {
+                "value": {
+                  "title": "PDP template value (e.g. Slug: /apple-magic-mouse/p OR /department/category/subcategory/)",
+                  "type": "string",
+                  "description": "The value can be: the PDP slug template (e.g. /apple-magic-mouse/p). A /department/category/subcategory/ template (e.g. /office/ OR /office/chairs/ OR /office/chairs/classic). If this field is empty this template will be the generic PDP."
```

Comment:
IMO we can improve these descriptions. Also the documentation should cover these details.